src/models.py

```diff
@@ -116,19 +118,19 @@ def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
                                        transformer, dropout=args.dropout,
                                        skip_embs=args.skip_embs,
                                        cove_layer=cove_layer,
-                                       sep_embs_for_skip=args.sep_embs_for_skip)
+                                       sep_elmo_embs_for_skip=args.sep_elmo_embs_for_skip,
+                                       sent_enc_type=args.sent_enc)
         log.info("Using Transformer architecture for shared encoder!")
     elif args.sent_enc == 'null':
         # Expose word representation layer (GloVe, ELMo, etc.) directly.
-        assert_for_log(args.skip_embs, f"skip_embs must be set for "
-                       "'{args.sent_enc}' encoder")
```

Comment:
This seems likely to cause confusion. The most intuitive null sentence encoder would just be a passthrough `f(x)=x`, for which a skip connection is useless. It seems like we're currently doing something much less helpful like `f(x)=null` instead.

(Again, we don't have to fix this here.)

Reply:
I think deleting this and the changes to `SentEncoder` address what Sam is saying (as in, these changes remove the confusion).